package.json

```
@@ -109,7 +109,7 @@
 		"@bartekbp/typescript-checkstyle": "5.0.0",
 		"@octokit/action": "5.0.2",
 		"@octokit/graphql": "5.0.5",
-		"@playwright/test": "1.32.3",
+		"@playwright/test": "^1.35.1",
```

We should stick to the fixed version:

```suggestion
		"@playwright/test": "1.35.1",
```

---

I don't have any strong opinion on this as this is a dev dependency. I will go ahead and change it to fixed version 🤝

---

I think that's the convention, but can't find the resource mentioning it, sorry. But I think we can assume that based on other dependencies 😅

---

Made the changes in [ec0c613](https://github.com/woocommerce/woocommerce-blocks/pull/10031/commits/ec0c6132c23b85559d7d1c1b5a08a015c6e2071f)